rolling/distribution.yaml

```
@@ -4832,6 +4832,22 @@ repositories:
       url: https://github.com/ros2/ros_testing.git
       version: rolling
     status: maintained
+  ros_tutorials:
+    doc:
+      type: git
+      url: https://github.com/ros/ros_tutorials.git
+      version: rolling
+    release:
+      tags:
+        release: release/rolling/{package}/{version}
+      url: https://github.com/ros2-gbp/ros_tutorials-release.git
+      version: 1.7.1-1
```

I think this is going to need to be:

```suggestion
    release:
      packages:
        - turtlesim
      tags:
        release: release/rolling/{package}/{version}
      url: https://github.com/ros2-gbp/ros_tutorials-release.git
      version: 1.7.1-1
```

(though I could use confirmation from @nuclearsandwich )